mappings/net/minecraft/world/SpawnHelper.mapping

```diff
@@ -124,16 +131,22 @@ CLASS net/minecraft/class_1948 net/minecraft/world/SpawnHelper
 		FIELD field_24398 cachedPos Lnet/minecraft/class_2338;
 		FIELD field_24399 cachedEntityType Lnet/minecraft/class_1299;
 		FIELD field_24400 cachedDensityMass D
+		FIELD field_34298 biome Lnet/minecraft/class_1959;
```

Comment:
same as in #2630, this is a
```suggestion
		FIELD field_34298 cachedBiome Lnet/minecraft/class_1959;
```
See how it is used with other `cachedXxx` fields in both of its usages.